pl_bolts/models/detection/faster_rcnn/faster_rcnn_module.py

```diff
@@ -46,38 +51,40 @@ def __init__(
         self,
         learning_rate: float = 0.0001,
         num_classes: int = 91,
+        backbone: Optional[str] = None,
+        fpn: bool = True,
         pretrained: bool = False,
         pretrained_backbone: bool = True,
         trainable_backbone_layers: int = 3,
-        replace_head: bool = True,
         **kwargs,
     ):
         """
         Args:
             learning_rate: the learning rate
             num_classes: number of detection classes (including background)
+            backbone: Pretained backbone CNN architecture.
+            fpn: If True, creates a FPN on top of Resnet based CNNs.
             pretrained: if true, returns a model pre-trained on COCO train2017
             pretrained_backbone: if true, returns a model with backbone pre-trained on Imagenet
             trainable_backbone_layers: number of trainable resnet layers starting from final block
         """
         super().__init__()
 
-        model = fasterrcnn_resnet50_fpn(
-            # num_classes=num_classes,
-            pretrained=pretrained,
-            pretrained_backbone=pretrained_backbone,
-            trainable_backbone_layers=trainable_backbone_layers,
-        )
+        self.learning_rate = learning_rate
+        self.num_classes = num_classes
+        self.backbone = backbone
+        if backbone is None:
+            self.model = fasterrcnn_resnet50_fpn(pretrained=pretrained,
+                                                 pretrained_backbone=pretrained_backbone,
+                                                 trainable_backbone_layers=trainable_backbone_layers)
```

Comment:
```suggestion
            self.model = fasterrcnn_resnet50_fpn(
                pretrained=pretrained,
                pretrained_backbone=pretrained_backbone,
                trainable_backbone_layers=trainable_backbone_layers,
            )
```
the same for the func bellow